sprint5/zadanie4.js

```diff
@@ -0,0 +1,25 @@
+/*Return the Last Element in an Array
+Create a function that accepts an array and returns the last item in the array.
+
+Examples
+getLastItem([1, 2, 3]) ➞ 3
+
+getLastItem(["cat", "dog", "duck"]) ➞ "duck"
+
+getLastItem([true, false, true]) ➞ true
+Notes
+Don't forget to return the result.
+If you get stuck on a challenge, find help in the Resources tab.
+If you're really stuck, unlock solutions in the Solutions tab.*/
+
+function getLastItem(arr) {
+  return arr[arr.length -1];
+}
+
+/* Tworzyłam funkcję "getLastItem", która przyjmuje jeden parametr "arr".
+Właśćiwość "length" zawiera liczbę elementów tablicy.
+Instrukcja return określa wartość zwracanej funkcji.
+Funkcja zwraca wartość ostatni indeks (3, "duck", true).
```

Comment:
`(3, "duck", true)` - to niepotrzebne, opisujemy ogólnie co robi funkcja
`Funkcja zwraca wartość ostatni indeks` - to nieprawda, funkcja nie zwraca indeksu.